package.json

```diff
@@ -21,14 +21,15 @@
     "css-in-js"
   ],
   "engines": {},
+  "main": "./dist/css-vendor.cjs.js",
+  "module": "./dist/css-vendor.esm.js",
+  "files": [
+    "dist"
```

Comment:
we also publish src and LICENSE in JSS core https://github.com/cssinjs/jss/blob/master/packages/jss/package.json#L22

Reply:
Src is only published for flow types, but this repo doesn't have them, so there is no need to publish the source code.